spec/functional/file_content_management/deploy_strategies_spec.rb

```diff
@@ -43,7 +43,7 @@ def normalize_mode(mode_int)
 
     ##
     # UNIX Context
-    let(:default_mode) { normalize_mode(0100666 - File.umask) }
```

Comment:
These are from another PR, I presume he has them on this one because he needs them for the tests to pass on his workstation.

See https://github.com/chef/chef/pull/3086